src/index.ts

```diff
@@ -120,10 +120,6 @@ program
                         Optionally, do not minify the bundle sent to the server.
                         Useful for debugging.`
     )
-    .option(
-        "-p, --toggle-public [togglePublic]",
-        "Toggle whether or not the block is public."
-    )
```

Comment:
@santios-volusion I think I probably did that wrong. We want to remove the documentation, but not the feature.